### ISSUE TITLE
Error when use field() in dataclass

### ISSUE THREAD

**Issue by theoqian** [pawamoy/mkdocstrings#227]
**Describe the bug**
I used field() in dataclass like this:
```
@dataclass
class MyData:
    arguments: Dict[str, int] = field(default_factory=dict)
```
And used `mkdocs serve` to create a docs website.
Then I got the follow error:
```
ERROR   -  mkdocstrings.extension: mismatched tag: line 93, column 599 (character p):
      <p>MyData(arguments:Dict[str,int]=<factory>)</p>
If your Markdown contains angle brackets < >, try to wrap them between backticks `< >`, or replace them with &lt; and &gt;
ERROR   -  Error reading page 'Docs/config.md': mismatched tag: line 93, column 599
```

**Information (please complete the following information):**
- OS: Ubuntu 18.04.4 LTS
- `mkdocstrings` version: 0.14.0

**Comment by malte-behrendt** [comment]
I've the very same problem.

OS: elementary OS 5.17 (Ubuntu 18.04 LTS-based)
mkdocstings version: 0.14.0

``INFO    -  Building documentation to directory: /workspaces/devops/tools/python/_redacted_/site
 WARNING -  mkdocstrings.handlers.python: Couldn't read source for '_redacted_.applications.application.Application.__eq__': could not get source code
 WARNING -  mkdocstrings.handlers.python: Couldn't read source for '_redacted_.applications.application.Application.__init__': could not get source code
 WARNING -  mkdocstrings.handlers.python: Couldn't read source for '_redacted_.applications.application.Application.__repr__': could not get source code
 WARNING -  mkdocstrings.handlers.python: Couldn't read source for '_redacted_.modules.chart.chart.Chart.__eq__': could not get source code
 WARNING -  mkdocstrings.handlers.python: Couldn't read source for '_redacted_.modules.chart.chart.Chart.__init__': could not get source code
 WARNING -  mkdocstrings.handlers.python: Couldn't read source for '_redacted_.modules.chart.chart.Chart.__repr__': could not get source code
 WARNING -  mkdocstrings.handlers.python: Couldn't read source for '_redacted_.modules.kube.kube.Kube.__eq__': could not get source code
 WARNING -  mkdocstrings.handlers.python: Couldn't read source for '_redacted_.modules.kube.kube.Kube.__init__': could not get source code
 WARNING -  mkdocstrings.handlers.python: Couldn't read source for '_redacted_.modules.kube.kube.Kube.__repr__': could not get source code
 ERROR   -  mkdocstrings.extension: mismatched tag: line 305, column 486 (character p):
 <p>Application(name: str, description: str = 'No description available', interface: str = 'restful-api', version: str = '0.0.0', language: Dict[str, str] = <factory>, source: Dict[str, Union[str, NoneType]] = <factory>, image: Dict[str, str] = <factory>, settings: Dict[str, Any] = <factory>, commands: Dict[str, Any] = <factory>, stages: Dict[str, Any] = <factory>, statics: List[str] = <factory>, dependencies: Dict[str, Any] = <factory>, packages: Dict[str, str] = <factory>)</p>
 If your Markdown contains angle brackets < >, try to wrap them between backticks `< >`, or replace them with &lt; and &gt;
 ERROR   -  Error reading page 'index.md': mismatched tag: line 305, column 486``

**Comment by pawamoy** [comment]
Hey @theoqian, @malte-behrendt, thank you for the report! It seems to be a duplicated of #222. As explained there, this is already fixed in master thanks to @oprypin's PR #209. If you want to try the master branch, please let us know if it works 🙂 
Since many users seem affected by this, I'll try to release the fix soon.
